client/src/component/AuthorInfoSection/index.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useEffect } from 'react';
+import {
+  Image, Button, InputGroup, Row, Col, FormControl,
+} from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import './AuthorInfoSection.css';
+
+const AuthorInfoSection = (props) => {
+  const [username, setUsername] = useState('');
+  const [fullName, setFullName] = useState('');
+  useEffect(() => {
+    setUsername(props.username);
+    setFullName(props.fullName);
+  }, [props.username, props.fullName]);
+  return (
+    <Row className="justify-content-center author-info-section">
+      {props.isLoggedIn ? (
+        <Col lg="4" md="8" xs="8" sm="8" className="text-center">
+          <Image
+            src={props.profilePic}
+            width={100}
+            height={100}
+            alt="Author"
+            className="author-image border border-primary text-center"
+            roundedCircle
+          />
+          <InputGroup>
+            <FormControl
+              plaintext
+              size="sm"
+              value={username}
+              onChange={(e) => setUsername(e.target.value)}
+              placeholder="Username"
+              aria-label="Username"
+              className="border-bottom input-box"
+            />
+          </InputGroup>
+          <InputGroup>
+            <FormControl
```

Comment:
You forget size prop in this form control. To prevent from these errors, you can create a component which will take onChange, aria-label, placeholder as inputs and it will return the input. In this way you will not forget props..

Reply:
:smiley: will surely implement this.